currency/l1/public.yml

```diff
@@ -159,3 +159,133 @@ paths:
             text/html:
               schema:
                 type: string
+
+  /data/validate:
+    post:
+      tags:
+        - Currency
```

Comment:
Maybe we should tag with "Data" for these?

Reply:
changed